tests/testthat/test-prepare-admin-panel.R

```diff
@@ -35,27 +35,27 @@ test_that("get_per_day_plot_data", {
   base <- data.frame(date = date_initial)
 
   per_day <- dplyr::tribble(
-    ~date,                 ~users, ~sessions, ~time, ~actions,
+    ~date,                 ~users, ~sessions, ~time, ~action,
     date_initial + 0,      3,         4,     0,        5,
     date_initial + 1,     31,        41,     8,       99
   )
 
   per_day_plot_data <- dplyr::tribble(
-    ~date,                 ~statistic, ~value, ~id,
-    date_initial + 0,    "logged users (unique)",      3,   3,
-    date_initial + 0,    "total opened sessions",      4,   1,
-    date_initial + 0, "avg session time (hours)",      0,   2,
-    date_initial + 0,  "total navigations and inputs",      5,   1,
+    ~date           , ~statistic                    , ~value, ~id, # nolint: commas_linter
+    date_initial + 0, "logged users (unique)"       ,      3,   1, # nolint: commas_linter
+    date_initial + 0, "total opened sessions"       ,      4,   1, # nolint: commas_linter
+    date_initial + 0, "avg session time (hours)"    ,      0,   2, # nolint: commas_linter
+    date_initial + 0, "total navigations and inputs",      5,   3, # nolint: commas_linter
     #
-    date_initial + 1,    "logged users (unique)",     31,   3,
-    date_initial + 1,    "total opened sessions",     41,   1,
-    date_initial + 1, "avg session time (hours)",      8,   2,
-    date_initial + 1,  "total navigations and inputs",     99,   1,
+    date_initial + 1, "logged users (unique)"       ,     31,   1, # nolint: commas_linter
+    date_initial + 1, "total opened sessions"       ,     41,   1, # nolint: commas_linter
+    date_initial + 1, "avg session time (hours)"    ,      8,   2, # nolint: commas_linter
+    date_initial + 1, "total navigations and inputs",     99,   3, # nolint: commas_linter
     #
-    date_initial + 2,    "logged users (unique)",      0,   3,
-    date_initial + 2,    "total opened sessions",      0,   1,
-    date_initial + 2, "avg session time (hours)",      0,   2,
-    date_initial + 2,  "total navigations and inputs",      0,   1,
+    date_initial + 2, "logged users (unique)"       ,      0,   1, # nolint: commas_linter
+    date_initial + 2, "total opened sessions"       ,      0,   1, # nolint: commas_linter
+    date_initial + 2, "avg session time (hours)"    ,      0,   2, # nolint: commas_linter
+    date_initial + 2, "total navigations and inputs",      0,   3, # nolint: commas_linter
```

Comment:
```suggestion
    date_initial + 2, "logged users (unique)",             0,   1,
    date_initial + 2, "total opened sessions",             0,   1,
    date_initial + 2, "avg session time (hours)",          0,   2,
    date_initial + 2, "total navigations and inputs",      0,   3,
```

Reply:
Thanks for the suggestion. I honestly can't believe this works 😂. So lint has issues only with spaces before commas but not after. I guess it makes sense since it also allows line breaks after commas.